org.omg.sysml.jupyter.kernel/src/main/java/org/omg/sysml/jupyter/kernel/ISysML.java

```diff
@@ -35,11 +35,18 @@
 
     private static volatile SysMLKernel kernel;
 
+    public static void initialize() {
+        System.setProperty("file.encoding", "UTF-8");
```

Comment:
> fixed this problem even without `file.encoding` fix in Windows 11, CP932 environment

Given this, I would ask that we consider reverting the changes in `ISysML` as I agree with the general sentiment that the Jupyter adaptation should be the thinnest layer required to interoperate with the real work being done on the Eclipse side.

Reply:
Let’s remove the setting of `file.encoding`, but leave the `initialization` method.

Reply:
I see.  It's done.